src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  .stars--4 > &__star:nth-child(-n + 4) {
```

Comment:
you should work with all possible `stars` modifiers (not only with `4`). so, just reuse your solution from `stars` task you solved before